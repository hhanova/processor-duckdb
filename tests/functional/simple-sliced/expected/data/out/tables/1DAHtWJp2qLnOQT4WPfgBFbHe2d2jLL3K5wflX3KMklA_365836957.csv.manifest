{"destination":"in.c-keboola-ex-google-drive-744153.ss-academy-workbook-registration-form","incremental":false,
"columns":["Timestamp","Email_Address","Vase_cele_jmeno","Nazev_vaseho_Tribu__oddeleni","Role"]}