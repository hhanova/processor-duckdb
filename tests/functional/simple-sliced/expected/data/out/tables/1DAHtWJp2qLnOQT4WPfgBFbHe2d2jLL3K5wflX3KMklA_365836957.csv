3/10/2021 17:03:14,1,A,Digital Sales,,2023-04-01
3/12/2021 15:15:16,2,B,Data tribe,,2023-04-01
3/24/2021 12:49:15,3,c,Wealth Mangement,Data Specialist,2023-04-01
4/9/2021 11:21:30,4,d,TEST,Data Scientist,2023-04-01
4/13/2021 12:25:25,5,e,Wealth Management,Sr. Data Developer,2023-04-01
6/4/2021 17:35:58,6,f,Tribe Personalizace,Product Owner,2023-04-01
6/15/2021 8:53:08,g,g,Personalization,Platform Developer,2023-04-01
6/15/2021 9:28:48,h,h,Personalizace,Platform Developer,2023-04-01
6/15/2021 11:39:55,i,i,Personalizace - nástroje,Platform Developer,2023-04-01
6/16/2021 9:38:40,j,j,Personalizace / Nástroje a kanály,Platform Developer,2023-04-01
7/9/2021 11:57:59,k,k,"Tribe Data, squad DWH",Platform Developer,2023-04-01
7/27/2021 14:59:48,l,l,Housing tribe,Data Specialist,2023-04-01
7/29/2021 15:41:39,m,m,HR,Data Specialist,2023-04-01
8/2/2021 15:38:42,n,n,Housing/Digi Sales,Data Specialist,2023-04-01
8/5/2021 13:23:23,o,o,HR,Data Specialist,2023-04-01
8/18/2021 8:08:14,p,p,Řízení rizik na finančních trzích,Data Scientist,2023-04-01
8/19/2021 10:27:10,q,q,Digitální procesy Store,Platform Admin,2023-04-01
9/17/2021 14:14:39,r,r,Agile coaches,Product Owner,2023-04-01
10/6/2021 15:07:07,s,s,Personalizace,Data Scientist,2023-04-01
10/6/2021 15:11:42,t,t,Personalizace,Data Scientist,2023-04-01
